test/kitchen/test/integration/benchmark_dogstatsd/rspec/integrations_dogstatsd_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
```

Comment:
the `benchmark_dogstatsd` files aren't actually used, are they?

Reply:
Nope I'll remove them